### DISCUSSION TITLE
Version 0.14.0 has been released!

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-java#2507] [Q&A]
https://github.com/open-telemetry/opentelemetry-java/releases/tag/v0.14.0

Auto instrumentation should follow in the next couple of days.

**Comment by shivchelwa** [comment]
When will it be available on maven and javadoc.io?

**Reply by jkwatson** [reply]
I just uncovered an issue with the publishing. Will have to release a patch release to address it. Sorry for the inconvenience.

**Reply by jkwatson** [reply]
v0.14.1 has been released and it is available in maven central now.
